pallets/fragments/src/lib.rs

```diff
@@ -274,8 +274,12 @@ pub mod pallet {
 			// create vault account
 			// we need an existential amount deposit to be able to create the vault account
 			let vault = Self::get_vault_id(hash);
-			let min_balance = T::Currency::minimum_balance();
```

Comment:
What's the point of have the assosciated type `Currency` for the `Config` trait of `pallet_assets` anymore?

Reply:
anyways, this is not even a bug - just something redundant